src/stepfunctions/steps/integration_resources.py

```diff
@@ -26,5 +27,22 @@ class IntegrationPattern(Enum):
     RequestResponse = ""
 
 
+def get_service_integration_arn(service, api, integration_pattern=IntegrationPattern.RequestResponse):
+
+    """
+    ARN builder for task integration
+    Args:
+        service(str): name of the task resource service
+        api(<Service>Api): api to be integrated of the task resource service
+        integration_pattern(IntegrationPattern, optional): integration pattern for the task resource.
+                                            Default as request response.
```

Comment:
Please apply the suggestion below. You can do this in the UI by clicking the "Commit suggestion" button.
```suggestion
        service (str): The service name for the service integration
        api (str): The api of the service integration
        integration_pattern (IntegrationPattern, optional): The integration pattern for the task. (Default: IntegrationPattern.RequestResponse)
```